sql.txt

```diff
@@ -1,14 +1,29 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
 
+SELECT CustomerName, ContactName, Country FROM [Customers] ORDER BY Country LIMIT 10 OFFSET 10;
+
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
 
+SELECT * FROM [Customers] WHERE ContactName LIKE 'A%';
+
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
 
+SELECT * FROM [OrderDetails] WHERE ProductID = 51 AND Quantity > 10;
+
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
 
+INSERT INTO Products VALUES 
+(NULL, "product1", 26, 5, "1 box", 5),
```

Comment:
Note: this only works if you put them in the right order and fill in all columns. Often is it better to specify the columns in the statement so you don't have to remember the order.

What you did works fine but I wanted to give a tip for future inserts to help prevent mistakes.